src/regex.rs

```diff
@@ -4,6 +4,7 @@ use std::fmt;
 
 use self::regex::Regex;
 
+#[derive(Debug)]
```

Comment:
Do we want the debug representation to display the values of these private internal fields? Right now this will output the debug value of the `inner: Regex` field. We may well want that, but I think it's worth making that decision explicit. Are you ok with this @sebasmagri?

Reply:
Any comments on this ?

Reply:
I think this is ok. Let's merge this in.